### PR TITLE
[ETL-426] Add new datasets as Glue tables and update crawlers

### PR DESCRIPTION
**Pull request by philerooski**

### CODE REVIEW

**Review by rxu17 — APPROVED** [review]
LGTM!

File: src/glue/resources/table_columns.yaml [hunk 1]
```diff
@@ -1,4 +1,49 @@
 tables:
+  3DRotation_v1:
```
Comment (rxu17) [inline src/glue/resources/table_columns.yaml:2]: Remind me again - how do we find the data types of the fields here for BD? Just through the Glue Crawler or parsing through the production 3D rotation data in Synapse ourselves (which would have all of the expected fields in one file or do you have to look through multiple)?

File: src/glue/resources/dataset_crawler_assignments.yaml [hunk 2]
```diff
@@ -3,6 +3,10 @@ standard:
   - sharedSchema_v1
   - WeatherResult_v1
   - sharedSchema_for_Gradual_Onset_v1
+  - 3DRotation_v1
```
Comment (thomasyu888) [inline src/glue/resources/dataset_crawler_assignments.yaml:6]: Are we informed when new datasets are added by our collaborators or bridge?

File: src/glue/resources/table_columns.yaml (hunk 1 above)
Comment (philerooski) [reply src/glue/resources/table_columns.yaml:2]: Initially, we derive the data types from the JSON Schema provided for the assessment. These Glue table schemas can end up evolving beyond any single JSON Schema (for example, if the assessment developers put out a new revision of the 3DRotation assessment with an additional field and remove other fields), but that hasn't happened IRL yet.

File: src/glue/resources/dataset_crawler_assignments.yaml (hunk 2 above)
Comment (philerooski) [reply src/glue/resources/dataset_crawler_assignments.yaml:6]: Short answer: yes. AFAIK nothing's been formalized (whatever that would look like), but everyone involved recognizes that BD needs to be part of the QA process. Sonia has been our link between the assessment developers (Northwestern, in this case) and ourselves so far.

**Review by thomasyu888 — APPROVED** [review]
🔥 LGTM!

**Pull request merged**